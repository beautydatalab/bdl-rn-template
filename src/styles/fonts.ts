import globalColors, {themeColor} from 'src/styles/colors';

export const sizes = {
  tiny: 10,
  small: 12,
  medium: 14,
  large: 16,
  extraLarge: 24,
  huge: 40,
};

export const colors = {
  default: globalColors.black,
  disable: globalColors.gray500,
  content: globalColors.gray700,
  theme: themeColor,
};
