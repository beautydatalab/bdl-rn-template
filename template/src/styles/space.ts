export const margins = {
  tiny: 4,
  short: 8,
  medium: 16,
  large: 24,
  huge: 32,
};

export const paddings = {
  tiny: 4,
  short: 8,
  medium: 16,
  large: 24,
  huge: 32,
};
