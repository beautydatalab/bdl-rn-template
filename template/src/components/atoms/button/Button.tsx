import React from 'react';
import {StyleProp, TextStyle} from 'react-native';
import {colors, sizes} from 'src/styles/fonts';
import styled from 'styled-components/native';

interface Props {
  style?: StyleProp<TextStyle>;
  title: string;
  onPress: () => void;
}

const StyledButton = styled.Button`
  font-size: ${sizes.large}px;
  color: ${colors.content};
  font-weight: 400;
`;

export const Button = (props: Props) => <StyledButton {...props} />;
