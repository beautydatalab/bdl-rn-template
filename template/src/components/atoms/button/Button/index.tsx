import React from 'react';

import {StyleProp, TextStyle} from 'react-native';

import styled from 'styled-components/native';

import {colors, sizes} from 'src/styles/fonts';

interface Props {
  style?: StyleProp<TextStyle>;
  title: string;
  onPress: () => void;
}

const StyledButton = styled.Button`
  font-size: ${sizes.large}px;
  color: ${colors.content};
  font-weight: 400;
`;

export const Button = (props: Props) => <StyledButton {...props} />;
