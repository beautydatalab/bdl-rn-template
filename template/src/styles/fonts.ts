import {colors as globalColors, themeColor} from 'src/styles/colors';

export const sizes = {
  tiny: 12,
  small: 14,
  medium: 16,
  large: 20,
  extraLarge: 24,
  huge: 32,
  extraHuge: 40,
};

export const colors = {
  default: globalColors.black,
  disable: globalColors.gray500,
  content: globalColors.gray700,
  theme: themeColor,
};
