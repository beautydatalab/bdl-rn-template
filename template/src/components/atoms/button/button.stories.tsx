import React from 'react';

import {storiesOf} from '@storybook/react-native';
import styled from 'styled-components/native';

import {palette} from 'src/styles/colors';
import {margins, paddings} from 'src/styles/space';

import {Button} from './Button';

const CenterView = styled.ScrollView.attrs({
  contentContainerStyle: {
    justifyContent: 'center',
    alignItems: 'center',

    paddingVertical: paddings.huge,
  },
})`
  flex: 1;
`;

const MarginBottomButton = styled(Button)`
  margin-bottom: ${margins.short}px;
`;

storiesOf('button', module)
  .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
  .add('Default Button', () => (
    <>
      <MarginBottomButton
        onPress={() => console.log('???')}
        title={'Outline Button'}
      />
      <MarginBottomButton
        onPress={() => console.log('???')}
        title={'Outline Color Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        onPress={() => console.log('???')}
        title={'Outline Color Yellow Background Button'}
        titleColor={palette.red400}
        backgroundColor={palette.yellow500}
      />
      <MarginBottomButton
        onPress={() => console.log('???')}
        loading
        title={'Outline loading Button'}
      />
      <MarginBottomButton
        onPress={() => console.log('???')}
        loading
        title={'Outline loading Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        onPress={() => console.log('???')}
        disabled
        title={'Outline disabled Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        title={'Solid Button'}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        title={'Solid Color Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        title={'Solid Color Yellow Background Button'}
        titleColor={palette.red400}
        backgroundColor={palette.yellow500}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        loading
        title={'Solid loading Button'}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        loading
        title={'Solid loading Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        type={'solid'}
        onPress={() => console.log('???')}
        disabled
        title={'Solid disabled Button'}
      />
      <MarginBottomButton
        type={'clear'}
        onPress={() => console.log('???')}
        title={'Clear Button'}
      />
      <MarginBottomButton
        type={'clear'}
        onPress={() => console.log('???')}
        title={'Clear Red Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        type={'clear'}
        onPress={() => console.log('???')}
        loading
        title={'Clear loading Button'}
      />
      <MarginBottomButton
        type={'clear'}
        onPress={() => console.log('???')}
        loading
        title={'Clear loading Button'}
        titleColor={palette.red400}
      />
      <MarginBottomButton
        type={'clear'}
        onPress={() => console.log('???')}
        disabled
        title={'Clear disabled Button'}
      />
    </>
  ));
