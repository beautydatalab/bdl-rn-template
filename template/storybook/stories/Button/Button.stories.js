import {text} from '@storybook/addon-knobs';
import {storiesOf} from '@storybook/react-native';
import React from 'react';
import {Text} from 'react-native';

import CenterView from '../../../src/components/CenterView';
import Button from './index';

storiesOf('Button', module)
  .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
  .add('with text', () => (
    <Button onPress={() => console.log('???')}>
      <Text>{text('Button text', 'Hello Button')}</Text>
    </Button>
  ))
  .add('with some emoji', () => (
    <Button onPress={() => console.log('???')}>
      <Text>😀 😎 👍 💯</Text>
    </Button>
  ));
