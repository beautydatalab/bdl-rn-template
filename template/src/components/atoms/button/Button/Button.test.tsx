import React from 'react';

import renderer from 'react-test-renderer';

import {Button} from 'src/components/atoms/button/Button';
import {palette} from 'src/styles/colors';

test('render correctly', () => {
  const tree = renderer
    .create(
      <>
        <Button onPress={() => console.log('???')} title={'Outline Button'} />
        <Button
          onPress={() => console.log('???')}
          title={'Outline Color Button'}
          titleColor={palette.red400}
        />
        <Button
          onPress={() => console.log('???')}
          title={'Outline Color Yellow Background Button'}
          titleColor={palette.red400}
          backgroundColor={palette.yellow500}
        />
        <Button
          onPress={() => console.log('???')}
          loading
          title={'Outline loading Button'}
        />
        <Button
          onPress={() => console.log('???')}
          loading
          title={'Outline loading Button'}
          titleColor={palette.red400}
        />
        <Button
          onPress={() => console.log('???')}
          disabled
          title={'Outline disabled Button'}
          titleColor={palette.red400}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          title={'Solid Button'}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          title={'Solid Color Button'}
          titleColor={palette.red400}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          title={'Solid Color Yellow Background Button'}
          titleColor={palette.red400}
          backgroundColor={palette.yellow500}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          loading
          title={'Solid loading Button'}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          loading
          title={'Solid loading Button'}
          titleColor={palette.red400}
        />
        <Button
          type={'solid'}
          onPress={() => console.log('???')}
          disabled
          title={'Solid disabled Button'}
        />
        <Button
          type={'clear'}
          onPress={() => console.log('???')}
          title={'Clear Button'}
        />
        <Button
          type={'clear'}
          onPress={() => console.log('???')}
          title={'Clear Red Button'}
          titleColor={palette.red400}
        />
        <Button
          type={'clear'}
          onPress={() => console.log('???')}
          loading
          title={'Clear loading Button'}
        />
        <Button
          type={'clear'}
          onPress={() => console.log('???')}
          loading
          title={'Clear loading Button'}
          titleColor={palette.red400}
        />
        <Button
          type={'clear'}
          onPress={() => console.log('???')}
          disabled
          title={'Clear disabled Button'}
        />
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
