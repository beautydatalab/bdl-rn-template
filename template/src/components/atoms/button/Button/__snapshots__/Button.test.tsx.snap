// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
Array [
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#43bc94",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#43bc94",
            },
          ],
          undefined,
          Object {
            "color": "#43bc94",
          },
        ]
      }
    >
      Outline Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#f96c70",
            },
          ],
          undefined,
          Object {
            "color": "#f96c70",
          },
        ]
      }
    >
      Outline Color Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#fae100",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#f96c70",
            },
          ],
          undefined,
          Object {
            "color": "#f96c70",
          },
        ]
      }
    >
      Outline Color Yellow Background Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#43bc94",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#43bc94"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#f96c70"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#909090",
        "borderRadius": 8,
        "borderWidth": 1,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#909090",
            },
          ],
          undefined,
          Object {
            "color": "#909090",
          },
        ]
      }
    >
      Outline disabled Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#43bc94",
        "borderColor": "#ffffff",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#ffffff",
            },
          ],
          undefined,
          Object {
            "color": "#ffffff",
          },
        ]
      }
    >
      Solid Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#43bc94",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#f96c70",
            },
          ],
          undefined,
          Object {
            "color": "#f96c70",
          },
        ]
      }
    >
      Solid Color Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#fae100",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#f96c70",
            },
          ],
          undefined,
          Object {
            "color": "#f96c70",
          },
        ]
      }
    >
      Solid Color Yellow Background Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#43bc94",
        "borderColor": "#ffffff",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#ffffff"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#43bc94",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#f96c70"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#909090",
        "borderColor": "#ffffff",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#ffffff",
            },
          ],
          undefined,
          Object {
            "color": "#ffffff",
          },
        ]
      }
    >
      Solid disabled Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#43bc94",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#43bc94",
            },
          ],
          undefined,
          Object {
            "color": "#43bc94",
          },
        ]
      }
    >
      Clear Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#f96c70",
            },
          ],
          undefined,
          Object {
            "color": "#f96c70",
          },
        ]
      }
    >
      Clear Red Button
    </Text>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#43bc94",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#43bc94"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#f96c70",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#f96c70"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Object {},
        ]
      }
    />
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#909090",
        "borderRadius": 8,
        "borderWidth": 0,
        "opacity": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#212121",
          },
          Array [
            Object {
              "color": "#909090",
            },
          ],
          undefined,
          Object {
            "color": "#909090",
          },
        ]
      }
    >
      Clear disabled Button
    </Text>
  </View>,
]
`;
